event/batch_test.go

```diff
@@ -31,7 +30,7 @@ func TestIsEmpty(t *testing.T) {
 
 		Convey("When the batch has a message added", func() {
 
-			batch.Add(ctx, message)
+			_ = batch.Add(ctx, message)
```

Comment:
Even in your tests, it's important to test the error returned is nil in a successful event
```suggestion
			err := batch.Add(ctx, message)
			Convey("Then the batch is not empty", func() {
				So(batch.IsEmpty(), ShouldBeFalse)
				
			        Convey("And no error is returned", func() {
				        So(err, ShouldBeNil)
			        })
			})
```